### PR TITLE
updated

### PR DESCRIPTION
**Pull request by CharlyCee**
course header removed and Navigation updated

### CODE REVIEW

File: front-end/src/App.js [hunk 1]
```diff
@@ -26,6 +28,8 @@ const App = () => {
 						<Route path="/course-details" element={<CourseDetails />} />
 						<Route path="/register/:courseName" element={<RegisterCourse />} />
 						<Route path="/user" element={<UserInformationScreen />} />
+						<Route path="/Login" element={<Login />} />
```
Comment (dellydel) [inline front-end/src/App.js:31]: I don't think there is a need to include the Login or Landing Page components, they are not yet being used anywhere and so we don't need a URL to access them.

**Review by dellydel — CHANGES_REQUESTED** [review]
Looks good after the unnecessary URLs are removed.
Unless there is a reason for them that I forgot about

File: front-end/src/App.js (hunk 1 above)
Comment (CharlyCee) [reply front-end/src/App.js:31]: yes, thats true.

File: front-end/src/App.js [hunk 2]
```diff
@@ -7,6 +7,8 @@ import Home from "./pages/Home";
 import Layout from "./pages/Layout";
 import RegisterCourse from "./pages/RegisterCourse";
 import UserInformationScreen from "./pages/UserInformationScreen";
+import Login from "./components/Login";
+import LandingPage from "./components/LandingPage";
```
Comment (BorisLaboratory) [inline front-end/src/App.js:11]: I think that the imports of Login and LandingPage should also be removed since not being used

**Pull request closed without merging**